lib/cooked_post_processor.rb

```diff
@@ -76,8 +77,6 @@ def post_process_images
       limit_size!(img)
       convert_to_link!(img)
     end
-
-    update_post_image
```

Comment:
If there isn't a good reason to put it inside the mutex, I would prefer not to make this change since we don't know why it was taken out of the mutex in the first place.